core/src/main/java/org/infinispan/topology/LocalUUID.java

```diff
@@ -0,0 +1,76 @@
+package org.infinispan.topology;
+
+import java.util.UUID;
+
+import org.infinispan.remoting.transport.Address;
+
+/**
+ * LocalUUID. A special {@link Address} UUID whose purpose is to remain unchanged across node
```

Comment:
I don't think you need to say the class name in the javadoc :)

TBH `Local` doesn't seem relevant either, perhaps `PersistentUUID` would sound better?